src/chrome/content/rules/SonarSource.com.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+	See also:
+		SonarCloud.io.xml
+		SonarLint.org.xml
+		SonarQube.org.xml
```

Comment:
Please only include a complete list in one file and always refer to the same list.